### DISCUSSION TITLE
.prose from the Tailwind Typography plugin doesn't inherit Daisy's theme colors

### DISCUSSION THREAD

**Discussion by mixmav** [saadeghi/daisyui#1115] [Q&A] [answered]
Hi all, on installing the @tailwindcss/typography plugin, and using the following markup, the output doesn't inherit Daisy's theme colors. Am I missing something?

```
<div class="prose">
    <h1>Garlic bread with cheese: What the science tells us</h1>
    <p>For years parents have espoused the health benefits of eating garlic bread with cheese to their children, with the food earning such an iconic status in our culture that kids will often dress up as warm, cheesy loaf for Halloween.</p>
    <p>But a recent study shows that the celebrated appetizer may be linked to a series of rabies cases springing up around the country.</p>
</div>
```

My `tailwind.config.js`

```
/** @type {import('tailwindcss').Config} */
module.exports = {
  content: [
	"./resources/**/*.{vue,js,ts,jsx,tsx}",
  ],
  
  theme: {
    extend: {},
  },

  plugins: [
    require('@tailwindcss/typography'),
    require("daisyui")
  ],

  daisyui: {
    prefix: "daisy-",
    darkTheme: "dracula"
  },
}
```

![image](https://user-images.githubusercontent.com/12530941/189249318-731e3e78-8d2c-40cb-9137-a2618b72364c.png)

**Comment by saadeghi** [comment, marked as answer]
Add `daisy-prose` class name to `prose`.  
```html
<div class="prose daisy-prose">
```
You're using `prefix` option so **ALL** class names coming form daisyUI will get a prefix. So you need `prose` class to get styles form typography plugin and also `daisy-prose` class to get colors from daisyUI

Let me know if you have any questions.
